src/lib/components/inventory/list_item_modal_styles.ts

```diff
@@ -0,0 +1,570 @@
+import {css} from 'lit';
+import {FloatElement} from '../custom';
+
+export const listItemModalStyles = [
+    ...FloatElement.styles,
+    css`
+        :host {
+            --scrollbar-width: ${window.innerWidth - document.documentElement.clientWidth}px;
+        }
+
+        .modal-backdrop {
+            position: fixed;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background: rgba(0, 0, 0, 0.6);
+            z-index: 1000;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            opacity: 0;
+            animation: fadeIn 0.2s ease forwards;
+        }
+
+        @keyframes fadeIn {
+            from {
+                opacity: 0;
+                backdrop-filter: blur(0px);
+                -webkit-backdrop-filter: blur(0px);
+            }
+            to {
+                opacity: 1;
+                backdrop-filter: blur(4px);
+                -webkit-backdrop-filter: blur(4px);
+            }
+        }
+
+        @keyframes fadeOut {
+            from {
+                opacity: 1;
+                backdrop-filter: blur(4px);
+                -webkit-backdrop-filter: blur(4px);
+            }
+            to {
+                opacity: 0;
+                backdrop-filter: blur(0px);
+                -webkit-backdrop-filter: blur(0px);
+            }
+        }
+
+        @keyframes modalIn {
+            from {
+                opacity: 0;
+                transform: scale(0.95);
+            }
+            to {
+                opacity: 1;
+                transform: scale(1);
+            }
+        }
+
+        @keyframes modalOut {
+            from {
+                opacity: 1;
+                transform: scale(1);
+            }
+            to {
+                opacity: 0;
+                transform: scale(0.95);
+            }
+        }
+
+        .modal-backdrop.closing {
+            animation: fadeOut 0.2s ease forwards;
+        }
+
+        .modal-content {
+            background: rgba(21, 23, 28, 0.8);
```

Comment:
Adding a `backdrop-filter: blur(16px)` will make it easier to see the text and match CSFloat FE a bit better.